packages/default/scss/dock-manager/_layout.scss

```diff
@@ -0,0 +1,230 @@
+        // A fix will be shipped likely in milestone 118 and then we can revisit this
+        @supports  (not (-ms-ime-mode: none)) and (not (overflow: -webkit-marquee)) and (not (-moz-appearance:none)) {
+            .k-toolbar-button {
+                padding-block: $kendo-button-padding-x;
+                padding-inline: $kendo-button-padding-y;
+            }
+            .k-toolbar-button .k-button-text {
+                writing-mode: vertical-lr;
+            }
+        }
+    }
+
+    // Panes
+    .k-dock-manager-pane-container {
+        width: 100%;
+        height: 100%;
+        display: flex;
+        flex-grow: 1;
+        overflow: hidden;
+    }
+
+    // Splitter
+    .k-dock-manager-splitter,
+    .k-dock-manager-unpinned-container {
+        height: 100%;
+        border-width: 0;
+        align-items: stretch;
+
+        > .k-pane {
+            height: auto;
+            flex-direction: column;
+            border-color: inherit;
+        }
+
+        .k-pane-header {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            padding-block: $kendo-dock-manager-pane-header-padding-y;
+            padding-inline: $kendo-dock-manager-pane-header-padding-x;
+            border-block-end-width: $kendo-dock-manager-pane-header-border-width;
+            border-block-end-style: $kendo-dock-manager-pane-header-border-style;
+            border-color: inherit;
+        }
+
+        .k-pane-title {
+            padding-block: $kendo-dock-manager-pane-title-padding-y;
+            padding-inline: $kendo-dock-manager-pane-title-padding-x;
+            font-family: $kendo-dock-manager-pane-title-font-family;
+            font-size: $kendo-dock-manager-pane-title-font-size;
+            line-height: $kendo-dock-manager-pane-title-line-height;
+            font-weight: $kendo-dock-manager-pane-title-font-weight;
+            flex: 1 1 auto;
+            white-space: nowrap;
+            text-overflow: ellipsis;
+            overflow: hidden;
+        }
+
+        .k-pane-scrollable {
+            height: 100%;
+            display: flex;
+            flex-direction: column;
+            border-color: inherit;
+        }
+
+        .k-pane-content {
+            height: 100%;
+            padding-block: $kendo-dock-manager-pane-content-padding-y;
+            padding-inline: $kendo-dock-manager-pane-content-padding-x;
+            overflow: auto;
+            flex: 1;
+        }
+
+        .k-pane-actions {
+            display: flex;
+            flex-flow: row nowrap;
+            align-items: center;
+            justify-content: center;
+        }
+
+        .k-pane-tabbed {
+            padding-block-start: $kendo-dock-manager-tabbed-pane-padding-y;
+            padding-block-end: $kendo-dock-manager-tabbed-pane-padding-x;
+            padding-inline: $kendo-dock-manager-tabbed-pane-padding-x;
+        }
+
+        .k-splitbar {
+            border-width: 0 1px;
```

Comment:
Maybe we could use a variable for the splitbar border width instead of hardcoding it?

Reply:
In general, overriding the splitbar here is not the best solution. This needs to be extracted in the Splitter styles and we should not apply any modifications in other components. Taking this into account, for the time being it is better to not expose variables for customizing this override.